src/services/milkomedaInfo.ts

```diff
@@ -0,0 +1,75 @@
+import { Request, Response } from "express";
+import config from "config";
+
+enum NetworkNames { // exemplary names here
+  testnet = "testnet",
+  mainnet = "mainnet",
+}
+
+type NetworkToken = {
+  name: NetworkNames,
+  tokens: Token[]
+}
+
+type Token = {
+    name: string
```

Comment:
best to specify the encoding of these strings (hex? ascii?)

I noticed you added it to the README but only for policyId and not for name

Reply:
Good point - I didn't think of encoding Token name (thought it's going to have token's name as a plain string). I'll add some hex info then